cypress/utils/globals.js

```diff
@@ -8,6 +8,17 @@ const CATEGORIES = {
   Performance: ['performance'],
 };
 
+// TODO remove if unused
```

Comment:
since it is unused, then can we remove it? 😁

Reply:
done